src/blue.ts

```diff
@@ -220,15 +228,11 @@ export function blueProxyFactory(env: MembraneBroker) {
         }
         getOwnPropertyDescriptor(shadowTarget: BlueShadowTarget, key: PropertyKey): PropertyDescriptor | undefined {
             const { target } = this;
-            let blueDesc = ReflectGetOwnPropertyDescriptor(shadowTarget, key);
```

Comment:
this is really the meat of the PR, we were using, in some cases, the value from the shadowTarget, now we only use values from the target, and the shadowTarget is only really use to satisfy the object invariants.